client/ayon_houdini/plugins/create/create_FBX.py

```diff
@@ -1,32 +1,34 @@
 # -*- coding: utf-8 -*-
-"""Creator for Unreal Static Meshes."""
+"""Creator plugin for creating Model product type as FBX."""
 from ayon_houdini.api import plugin
 from ayon_core.lib import BoolDef, EnumDef
 
 import hou
 
 
-class CreateStaticMesh(plugin.HoudiniCreator):
-    """Static Meshes as FBX. """
-
-    identifier = "io.openpype.creators.houdini.staticmesh.fbx"
-    label = "Static Mesh (FBX)"
-    product_type = "staticMesh"
-    icon = "fa5s.cubes"
+class CreateFBX(plugin.HoudiniCreator):
+    """Model as FBX. """
 
+    identifier = "io.openpype.creators.houdini.modelFBX.fbx"
```

Comment:
No need for `fbx` twice in the identifier. Also, since this is a 'new' creator - let's make sure the identifier is also using the newer `ayon` instead of legacy `openpype` name.
```suggestion
    identifier = "io.ayon.creators.houdini.model.fbx"
```